ucs/clientlist.cpp

```diff
@@ -594,6 +596,35 @@ void Clientlist::CheckForStaleConnections(Client *c) {
 	}
 }
 
+std::string RemoveDuplicateChannels(std::string InChannels) {
+	// Split the string by ", " and store the names in a vector
```

Comment:
We have `Strings::Split` for this exact functionality

Reply:
Fixed